utils/virtcontainers-setup.sh

```diff
@@ -59,7 +59,7 @@ cp -r ${busybox_bundle}/* ${pause_bundle}/
 echo -e "Move to ${tmpdir}/${virtcontainers_build_dir}"
 pushd ${tmpdir}/${virtcontainers_build_dir}
 echo "Clone cni"
-git clone https://github.com/containernetworking/cni.git
+git clone https://github.com/containernetworking/plugins.git
```

Comment:
Vendoring the script would have made sense if they didn't move the plugins to a different repo. But now that they are in a separate one, there is no way to track a kind of synchronization between both repos (cni and plugins).